cmd/armada-load-tester/cmd/root.go

```diff
@@ -17,19 +17,6 @@ func init() {
 var rootCmd = &cobra.Command{
 	Use:   "armada-load-tester command",
 	Short: "Command line utility to submit many jobs to armada",
-	Long: `
```

Comment:
Why did we remove the Long description?

Reply:
Oops thanks fo noticing, I only wanted to remove the `example structure`, let me correct that

Reply:
@zuqq has provided spaces and all in the same `example structure` that I'm trying to remove, I also commented on his PR but did'nt hear back from him. This deletion is not vital to the project its just that of you take a look at any other command-line tool eg: `kubectl` you won't any kind of structure, you only see what kubectl does and its command so only for the purpose to make `armadactl` look good I have proposed this change